scss/_tables.scss

```diff
@@ -63,6 +63,13 @@
     width: $spacer * 2;
     max-width: $spacer * 2;
   }
+
+  // Use negative margins on icons in tables to preserve row height
+  svg,
+  img {
+    margin-top: $table-cell-icon-margin-top;
```

Comment:
TBH I was a bit afraid when I read it on how the rendering would be with bigger images. But one can disable the default behavior by doing this so I suppose it's OK: `<svg class="... my-0" width="5rem" height="5rem" .../>`. Here's the rendering FYI:

![Screenshot 2023-04-05 at 08 17 17](https://user-images.githubusercontent.com/17381666/229996902-78fe0fe2-80b8-44b7-8f6e-6eafd3343e07.png)